app/js/views/errorView.js

```diff
@@ -33,7 +33,7 @@ export default class ErrorView extends View {
     }
 
     this.message = response.statusText;
-    if (response.hasOwnProperty('responseJSON') && response.responseJSON !== (null || undefined)) {
+    if (response.hasOwnProperty('responseJSON') && response.responseJSON !== (null && undefined)) {
```

Comment:
This doesnt check if response.responseJSON is undefined.
try: 
`if (response.hasOwnProperty('responseJSON') && response.responseJSON)`